sdk/rust/oak/src/stubs.rs

```diff
@@ -49,6 +49,9 @@ pub extern "C" fn wait_on_channels() {
 pub extern "C" fn channel_read() {
     panic!("stub function invoked!");
 }
+pub extern "C" fn channel_read_with_downgrade() {
```

Comment:
This is missing the `#[no_mangle]` so doesn't work as a stub, btw.

Reply:
Updated in https://github.com/project-oak/oak/pull/1839